ci-operator/step-registry/osd-ccs/fleet-manager/sc/deprovision/osd-ccs-fleet-manager-sc-deprovision-ref.yaml

```diff
@@ -0,0 +1,21 @@
+ref:
+  as: osd-ccs-fleet-manager-sc-deprovision
+  from_image:
+    namespace: ci
+    name: cli-ocm
+    tag: latest
+  grace_period: 3h10m
```

Comment:
grace_period is the time used to shut down the pod gracefully after the pod finishes its work. `3h10m` is a too big number, we should not wait for so long times,`10m` is enough.

Reply:
thanks a lot! learning day!